cabby/geo/regions.py

```diff
@@ -32,8 +34,15 @@ def get_region(region: Text):
     polygon_area = box(
             miny=40.425, minx=-80.035, maxy=40.460, maxx=-79.930,
             ccw=True)
-  else: # D.C
-    polygon_area = box(
-        miny=38.90821, minx=-77.04053, maxy=38.90922, maxx=-77.03937,
-        ccw=True)
-  return polygon_area  
\ No newline at end of file
+  elif region == "DC": # D.C
```

Comment:
Where is the `if` parent to this `elif`?

Reply:
line 29  if region == "Manhattan":